package.json

```diff
@@ -1251,7 +1251,7 @@
     "pretty-ms": "6.0.0",
     "prop-types": "^15.8.1",
     "proxy-from-env": "1.0.0",
-    "puppeteer": "24.4.0",
+    "puppeteer": "24.6.1",
```

Comment:
Do we always lock in a specific version of `puppeteer`? Would there be benefit to allowing a range, or does this cause problems?

Reply:
I believe that's because we always want to pin to a specific Chromium version that is pinned to a specific Puppeteer version: https://github.com/puppeteer/puppeteer/blob/main/versions.json. But the team can keep me honest.

Reply:
@azasypkin this is correct.